javamelody-core/src/main/java/net/bull/javamelody/Parameter.java

```diff
@@ -371,6 +371,11 @@
 	 */
 	STATSD_ADDRESS("statsd-address"),
 
+	/**
+	 * Use `javamelody.metricName` instead of `javameloady.context.hostName.metricName`. False by default.

```

Comment:
`Prefix of StatsD metric names. javamelody.context.hostName.metricName by default.` would be better.

Reply:
Fixed